🐛 Add environment variable for base URL and update metadata generation

FunamaYukina opened the pull request:
## Summary
<!-- Briefly describe the changes and the purpose of the PR. -->

In [the previous PR](https://github.com/liam-hq/liam/pull/514), the image path for the docs was not set up correctly, so I've added it now.

![ss 2717](https://github.com/user-attachments/assets/ee9f2bdb-7acc-4756-a17d-4f51651740dd)

Although I've updated the paths in this PR, the preview environment continues to use the production paths.💭

preview: https://liam-docs-3w1g3rdn9-route-06-core.vercel.app/docs

![ss 2718](https://github.com/user-attachments/assets/5ad664da-764a-4200-90f9-a75cc7e44ede)



## Related Issue
<!-- Mention the related issue number if applicable. -->

## Changes
<!-- List the main changes made in this PR. -->

## Testing
<!-- Briefly describe the testing steps or results. -->

## Other Information
<!-- Add any other relevant information for the reviewer. -->

MH4GF commented:
@FunamaYukina 
Q: Was this problem only in the development environment? Was it also in the production environment?

MH4GF commented:
@FunamaYukina OK, I understand that you set the environment variable from the Vercel dashboard.
I mistakenly thought the problem was in the development environment because only .env.development was added.

`NEXT_PUBLIC_BASE_URL` is not secret variables, How about adding .env.production and commit?

FunamaYukina commented:
@MH4GF 
My apologies🙇‍♀️. I overlooked mentioning that I've configured environment variables on Vercel. Given that this information isn't classified, I've created an ``env.production`` file as suggested.（I have removed the ``environment variable`` from Vercel.）

https://github.com/liam-hq/liam/pull/518/commits/3f05b5172e6425aa7e2840dc114d3d8f15ac9606

I've set default values as I determined that the ``env.development`` file was redundant.
Additionally, I've fixed the incorrect path to the env file.

MH4GF commented:
@FunamaYukina 
It's not a bad idea to set it up in Vercel!
But I think it would be easier to understand this if it were clearly stated in the code. Thanks!!

MH4GF reviewed (approved):
LGTM 👍🏻 Thanks for correcting!

FunamaYukina commented:
Thank you for your review!!✨
